hapi-proto/src/main/proto/TransactionBody.proto

```diff
@@ -117,5 +121,9 @@ message TransactionBody {
     TokenWipeAccountTransactionBody tokenWipe = 39; // Wipes amount of tokens from an account
     TokenAssociateTransactionBody tokenAssociate = 40; // Associate tokens to an account
     TokenDissociateTransactionBody tokenDissociate = 41; // Dissociate tokens from an account
+
+    ScheduleCreateTransactionBody scheduleCreation = 42; // Creates a schedule entity instance
+    ScheduleDeleteTransactionBody scheduleDelete = 43; // Deletes a schedule entity instance
+    ScheduleSignTransactionBody scheduleSign = 44; // Signs a schedule entity instance
```

Comment:
Maybe we could define that the entity is tx: `scheduled transaction instance` instead of `schedule entity instance`